src/NuGet.Clients/VsConsole/PowerShellHost/PowerShellHost.cs

```diff
@@ -330,14 +337,19 @@ private void UpdateWorkingDirectory()
             if (Runspace.RunspaceAvailability == RunspaceAvailability.Available)
             {
                 // if there is no solution open, we set the active directory to be user profile folder
-                string targetDir = _solutionManager.IsSolutionOpen ?
-                    _solutionManager.SolutionDirectory :
-                    Environment.GetEnvironmentVariable("USERPROFILE");
+                string targetDir = GetSolutionDirectory();
 
                 Runspace.ChangePSDirectory(targetDir);
             }
         }
 
+        private string GetSolutionDirectory()
+        {
+            return _solutionManager.IsSolutionOpen ?
+                _solutionManager.SolutionDirectory :
+                Environment.GetEnvironmentVariable("USERPROFILE");
```

Comment:
could this just use string.empty instead of userprofile? I don't see this path actually being used